DataMigration/src/test/java/org/sagebionetworks/dian/datamigration/HmDataModelTests.java

```diff
@@ -231,7 +234,8 @@ public void test_findDeviceId() {
 
         deviceId = HmDataModel.TableRow.findParticipantDeviceId("2", userDeviceIdList);
         assertNotNull(deviceId);
-        assertEquals("abc5cbaf-288c-48dd-9d4a-98c90213ac01", deviceId.device_id);
+        // There are two matches for that participant, but this is the most recent
+        assertEquals("def5cbaf-288c-48dd-9d4a-98c90213ac01", deviceId.device_id);
```

Comment:
Since the two records have the same ID and time stamp how does this test make sure the right one is returned?

Reply:
The timestamps are different for both entries in the list that have participant id "2".

Reply:
Should this assertion be checking that the later timestamp is returned, i.e.:
``` 
assertEquals("1586165222", deviceId.created_at);

```
?